### PR TITLE
update build desktop docs

### PR DESCRIPTION
**Pull request by dmitryn**
Update build desktop docs after switch to `clj-rn` tool https://github.com/status-im/status-react/pull/6485

### CODE REVIEW

File: source/docs/build_desktop.md [hunk 1]
```diff
@@ -49,17 +49,16 @@ npm update
...
 1. Run the following commands:
     ``` bash
     git clone https://github.com/status-im/status-react.git
     cd status-react
-    make dev-desktop
     ```
 1. In separate terminal tab: `npm start` (note: it starts react-native packager )
 1. In separate terminal tab: `node ./ubuntu-server.js`
-1. In separate terminal tab: `lein figwheel-repl desktop` (note: wait until sources are compiled)
+1. In separate terminal tab: `clj -R:dev build.clj watch --platform desktop` (note: wait until sources are compiled)
```
[the remark this answers is not in the document]
Comment (dmitryn) [reply source/docs/build_desktop.md]: @siphiuel you still might need Leiningen to run `lein prod-build` so i would leave it as a requirement. Also, i will add `clj` install notes.

Comment (siphiuel) [inline source/docs/build_desktop.md]: Also: `make prepare-desktop` step is necessary.

[the remark this answers is not in the document]
Comment (yenda) [reply source/docs/build_desktop.md]: also can't run `npm start` before `make prepare-desktop`

Comment (siphiuel) [reply source/docs/build_desktop.md]: Just noticed that the builds are compiled into a wrong location:
```
Successfully compiled build :desktop to "target/ios/desktop.js" in 49.355 seconds.
```

[the remark this answers is not in the document]
Comment (yenda) [reply source/docs/build_desktop.md]: and shouldn't `clj -R:dev build.clj watch --platform desktop` be replaced by `make watch-desktop`?

Comment (dmitryn) [reply source/docs/build_desktop.md]: @siphiuel good catch, found the typo https://github.com/status-im/status-react/pull/6520/files

[the remark this answers is not in the document]
Comment (dmitryn) [reply source/docs/build_desktop.md]: @yenda good idea, will replace

Comment (siphiuel) [reply source/docs/build_desktop.md]: hahaha, https://github.com/status-im/status-react/pull/6519 )) Closing this now

**Comment by dmitryn** [comment]
@siphiuel @yenda updated the docs, please take a look

**Comment by dmitryn** [comment]
```
Merging is blocked
The base branch restricts merging to authorized users.
```

@yenda any idea how to merge it?

**Comment by yenda** [comment]
@dmitryn I suppose @andytudhope has the rights, maybe @jeluard

**Pull request merged**